/armor_detector:
  ros__parameters:
    use_depth: false
    subscribe_compressed: false

    min_lightness: 160

    light:
      max_angle: 40.0
      max_ratio: 0.55
      min_ratio: 0.1

    armor:
      max_angle: 35.0
      max_center_ratio: 1.6
      min_center_ratio: 0.4
      min_light_ratio: 0.6
